experiments/temporal_robustness/run.py

```
@@ -0,0 +1,1213 @@
+                overall_status=overall,
+            )
+    return RunResult(overall_status=overall, skipped=False, output_dir=output_dir)
+
+
+def _run_item_with_retries(item: RunItem, batch: BatchSettings) -> RunResult:
+    attempts = batch.max_retries + 1
+    last_result: RunResult | None = None
+    for attempt in range(1, attempts + 1):
+        try:
+            result = _run_item(item)
+        except BaseException as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `BaseException`

(BLE001)